website_multi_theme/wizards/website_config_settings_view.xml

```diff
@@ -15,7 +15,7 @@
             <field name="module_website_form_editor" position="after">
                 <label for="multi_theme_id"/>
                 <div>
-                    <field name="multi_theme_id" class="oe_inline"/>
+                    <field name="multi_theme_id" class="oe_inline" required="1"/>
```

Comment:
Why not require it in the model?

Reply:
Because we have default website, which doesn't have multi_theme_id on installation and we cannot set it on installation, because *Default theme* is created after loading python files

Reply:
It makes sense. Could you add a note here then please? It's not obvious...